.github/workflows/container-image.yml

```diff
@@ -0,0 +1,137 @@
+name: Build Container Image
+
+concurrency:
+  group: ci-${{ github.run_id }}
+  cancel-in-progress: true
+
+on:
```

Comment:
Could you add a `workflow_dispatch` event trigger here so we can manually trigger rebuilds? We'll probably want to do that on occasion since the container build just pulls from the `:latest` base image.